Expand VS Code extension overview (#7901)
circleci/circleci-docs: jekyll/_cci2/introduction-to-yaml-configurations.adoc

If you would like a more complex configuration tutorial, visit the <<config-intro#,Configuration Tutorial>> page. Before starting this tutorial, you should already have a CircleCI account set up, as you will follow along in the CircleCI web UI. You can also find a variety of configuration examples on the <<sample-config#, Sample Configuration>> page.

[#visual-studio-code-extension]
## VS Code extension

> If you are a VS Code user, you might find the official xref:vs-code-extension-overview#[CircleCI extension] useful when writing, editing, navigating and troubleshooting your YAML Config files.

Reviewer: ```suggestion
If you use VS Code, you might find the official xref:vs-code-extension-overview#[CircleCI extension] useful when writing, editing, navigating and troubleshooting your YAML Config files.
```

If you would like a more complex configuration tutorial, visit the <<config-intro#,Configuration Tutorial>> page. Before starting this tutorial, you should already have a CircleCI account set up, as you will follow along in the CircleCI web UI. You can also find a variety of configuration examples on the <<sample-config#, Sample Configuration>> page.

[#visual-studio-code-extension]

> ## VS Code extension

Author: Should we include a relevant screenshot/capture here, or just try and keep most images on the overview page?
